stack/clone.go

```diff
@@ -153,16 +152,14 @@ func updateStackID(stackdir string) error {
 
 	logger.Trace().Msg("finding file containing stack definition")
 
-	stackFilePath, body := getStackBody(parser)
-	if body == nil {
-		return errors.E("updating stack ID: stack block not found")
+	stackFilePath := getStackFilepath(parser)
```

Comment:
maybe call this `stackParsedFile`?

Reply:
hmm but it is not returning a parsed file :thinking: . Maybe stackBlockFilepath ? wanted to make it clear that it returns the filepath of where the stack block is defined. What is being "get" is kinda hard to explain with a small name :laughing:

Reply:
gotcha! I thought it was returning the hcl.File object.